client/template.js

```diff
@@ -11,10 +11,10 @@ module.exports = async(name, title = '', props = {})=>{
 		<link rel="icon" href="/assets/homebrew/favicon.ico" type="image/x-icon" />
 		<meta property="og:title" content="${props.brew?.title ?? 'Homebrewery - Untitled Brew'}">
 		<meta property="og:url" content="${HOMEBREWERY_PUBLIC_URL}/${props.brew?.shareId ? `share/${props.brew.shareId}` : ''}">
-		<meta property="og:image" content="${props.brew?.thumbnail ?? `${HOMEBREWERY_PUBLIC_URL}/thumbnail.png`}">
-		<meta property="og:description" content="${props.brew?.description ?? 'No description.'}">
+		<meta property="og:image" content="${props.brew?.thumbnail ? props.brew?.thumbnail : `${HOMEBREWERY_PUBLIC_URL}/thumbnail.png`}">
```

Comment:
If nullish coalescing broke it, you should be able to just use `||` instead of a whole ternary.

```suggestion
		<meta property="og:image" content="${props.brew?.thumbnail || `${HOMEBREWERY_PUBLIC_URL}/thumbnail.png`}">
```